--- firefox-66.0.3/browser/base/content/browser.js.orig	2019-04-22 16:11:47.908683607 +0200
+++ firefox-66.0.3/browser/base/content/browser.js	2019-04-22 16:20:26.249208849 +0200
@@ -6165,12 +6165,9 @@ function middleMousePaste(event) {
   }
 
   UrlbarUtils.getShortcutOrURIAndPostData(clipboard).then(data => {
-    try {
-      makeURI(data.url);
-    } catch (ex) {
-      // Not a valid URI.
-      return;
-    }
+    var URIFixup = Components.classes["@mozilla.org/docshell/urifixup;1"].getService(Components.interfaces.nsIURIFixup);
+    data.url = URIFixup.createFixupURI(data.url, 1).spec;
+    // 1 is FIXUP_FLAG_ALLOW_KEYWORD_LOOKUP
 
     try {
       UrlbarUtils.addToUrlbarHistory(data.url, window);
