--- firefox-70.0/modules/libpref/init/all.js~	2019-10-26 19:00:48.000000000 +0200
+++ firefox-70.0/modules/libpref/init/all.js	2019-10-26 19:04:38.462322031 +0200
@@ -5399,8 +5399,8 @@
 
     // Health Report is enabled by default on all channels.
     // Do note that the toggle on Fenix and Focus does NOT reflect to this pref.
-    pref("datareporting.healthreport.uploadEnabled", true);
-    pref("datareporting.usage.uploadEnabled", true);
+    pref("datareporting.healthreport.uploadEnabled", false);
+    pref("datareporting.usage.uploadEnabled", false);
   #endif
 #endif
 
