--- firefox-70.0/modules/libpref/init/all.js~	2019-10-26 19:00:48.000000000 +0200
+++ firefox-70.0/modules/libpref/init/all.js	2019-10-26 19:04:38.462322031 +0200
@@ -5399,7 +5399,7 @@
     pref("datareporting.healthreport.infoURL", "https://www.mozilla.org/legal/privacy/firefox.html#health-report");
 
     // Health Report is enabled by default on all channels.
-    pref("datareporting.healthreport.uploadEnabled", true);
+    pref("datareporting.healthreport.uploadEnabled", false);
   #endif
 #endif
 
