--- firefox-81.0.1/security/sandbox/linux/SandboxFilter.cpp.orig	2020-09-30 19:41:50.000000000 +0200
+++ firefox-81.0.1/security/sandbox/linux/SandboxFilter.cpp	2020-10-04 20:11:32.552050641 +0200
@@ -479,7 +479,13 @@
 
     switch (sysno) {
         // Timekeeping
+#ifdef __NR_clock_nanosleep_time64
+      case __NR_clock_nanosleep_time64:
+#endif
       case __NR_clock_nanosleep:
+#ifdef __NR_clock_getres_time64
+      case __NR_clock_getres_time64:
+#endif
       case __NR_clock_getres:
 #ifdef __NR_clock_gettime64
       case __NR_clock_gettime64:
@@ -513,6 +519,9 @@
         return Allow();
 
         // Thread synchronization
+#ifdef __NR_futex_time64
+      case __NR_futex_time64:
+#endif
       case __NR_futex:
         // FIXME: This could be more restrictive....
         return Allow();
@@ -523,6 +532,9 @@
       CASES_FOR_epoll_create:
       CASES_FOR_epoll_wait:
       case __NR_epoll_ctl:
+#ifdef __NR_ppoll_time64
+      case __NR_ppoll_time64:
+#endif
       CASES_FOR_poll:
         return Allow();
 
@@ -1095,6 +1107,9 @@
 #endif
 
       CASES_FOR_select:
+#ifdef __NR_pselect6_time64
+      case __NR_pselect6_time64:
+#endif
         return Allow();
 
       CASES_FOR_getdents:
