--- firefox-78.0/testing/geckodriver/src/capabilities.rs.orig	2020-07-01 11:46:43.489401648 +0200
+++ firefox-78.0/testing/geckodriver/src/capabilities.rs	2020-07-01 11:47:03.525900589 +0200
@@ -79,7 +79,7 @@
 
     fn version_from_binary(&self, binary: &PathBuf) -> Option<String> {
         let version_regexp =
-            Regex::new(r#"\d+\.\d+(?:[a-z]\d+)?"#).expect("Error parsing version regexp");
+            Regex::new(r#"[0-9]+\.[0-9]+(?:[a-z][0-9]+)?"#).expect("Error parsing version regexp");
         let output = Command::new(binary)
             .args(&["-version"])
             .stdout(Stdio::piped())
